### ISSUE TITLE
Review alt text for WCAG compliance on About page

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2612]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the About page.

#### Details
Developers identified that some alt texts in the About page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items 
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on at the bottom in the Resources/Instructions section of this issue where it says "Google sheet Tab for your page".
- [x] For each image on the page, identify if the alt text needs to be changed or not using the recourses below and fill out the google sheet. To locate the alt text use the Google Chrome browser, right click "inspect" to view pages code.
- [x] After completing auditing each image change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [ ] Add Ready for dev leads label
   - [ ] Move to New Issue Approval Column

Development
- [ ] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [ ] Close this issue

### Resources/Instructions
[About page](https://www.hackforla.org/about/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=2047679475)

**Comment by phuonguvan** [comment]
1. Progress: Finish 3 alt texts, waiting for review
2. Blockers: Confused if I am changing the alt of the title or information below it 
3. Availability: All week
4. ETA: 2 weeks

**Comment by phuonguvan** [comment]
1. Progress: Finish all alt texts, waiting for review
2. Blockers: Not sure if all of them are correct
3. Availability: All week
4. ETA:

**Comment by bishrfaisal** [comment]
This issue is ready for dev leads to make good first and second issues out of, with one exception. Please ignore the wins card under the Accomplishments subheading on the About page and do not create beginner issues on it.
![Capture](https://user-images.githubusercontent.com/112529990/195454493-84189ab6-5394-4341-9fd8-21a18e9aa3a1.JPG)
This is something that needs to be discussed in depth since it is an svg file and requires more understanding on how to properly write an alt text along with aria labels. Once issues on other alt texts are made, please send this issue back to QA and we will evaluate it again.

**Comment by 7kram** [comment]
All issues have been created for alt text. They can be found here:

- https://github.com/hackforla/website/issues/3361

**Comment by 7kram** [comment]
All issues have been created for alt text for this page so I'll go ahead and move it to the done column. You can refer to the comment above https://github.com/hackforla/website/issues/2612#issuecomment-1280006194

**Comment by ExperimentsInHonesty** [comment]
Has been added to the agenda.  Marking it for discussion dev and pm.  Moved it to the questions column

**Comment by ExperimentsInHonesty** [comment]
This svg for the wins card has been wrapped in a img src and is working fine.  I am moving this to done and removing from the dev pm agenda
